conans/client/conf/detect.py

```diff
@@ -5,7 +5,7 @@
 
 from conans.client.output import Color
 from conans.model.version import Version
-from conans.client.tools import latest_visual_studio_version_installed
+from conans.client.tools.win import _latest_visual_studio_version_installed
```

Comment:
Importing a private from other module, not the best.

Reply:
may be just remove output parameter from the original function and avoid private trampoline at all?